src/main/java/com/odysseusinc/arachne/datanode/dto/datasource/validation/DataSourceCredentialsValidator.java

```diff
@@ -30,13 +31,23 @@
 @Component
 public class DataSourceCredentialsValidator extends BaseValidator implements ConstraintValidator<ValidCredentials, DataSource> {
 
+    private static final String USERNAME_FIELDNAME = "username";
```

Comment:
Why do we have different username field names here and in `CreateDataSourceCredentialsValidator`?

Reply:
Well, this name is used to bind violation with concrete field, but field names are different for DTO and Entity object.